src/redux/configureStore.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable object-curly-newline */
+import { configureStore } from '@reduxjs/toolkit';
+import { combineReducers, applyMiddleware } from 'redux';
+import thunk from 'redux-thunk';
+import logger from 'redux-logger';
+import country from './Country/countries.js';
+import pollutionReducer from './Pollution/pollution.js';
+
+const rootReducer = combineReducers({
```

Comment:
It is against microverse policy to disable linter for linters help us write clean code please it would be nice if you enbaled esint in this file.

Reply:
@KayLemba Done👌